terraform/modules/daily_snapshot/service/upload_snapshot.sh

```diff
@@ -94,6 +105,7 @@ docker run \
   --rm \
   --user root \
   -v "$CHAIN_DB_DIR:/home/forest/forest_db":z \
+  -v "$CHAIN_LOGS_DIR:$CHAIN_LOGS_DIR":z \
```

Comment:
Using fixed mount points inside the container is preferable to avoid accidental clashes. How about putting the logs at `/home/forest/logs`?